### PR TITLE
update TCNJ to use CE2

### PR DESCRIPTION
**Pull request by coldbeer101**
CE1 is not longer used. Converted it to use CE2

### CODE REVIEW

**Comment by brianhlin** [comment]
Did CE1 ever run jobs? Did they run in the last year? If so, we should create a duplicate entry for CE2 instead of replacing the CE1 entry

**Comment by coldbeer101** [comment]
Looks like last jobs were May. So yes it has run jobs in the last year. Ill revert that and add a new one for CE2

File: topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml [hunk 1]
```diff
@@ -85,3 +85,30 @@ Resources:
           hidden: false
     Tags:
       - CC*
+  TCNJ-ELSA-CE2:
+    ID:
```
Comment (brianhlin) [inline topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:89]:
```suggestion
    ID: 1341
```

**Review by brianhlin — APPROVED** [review]
LGTM!

File: topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml [hunk 2]
```diff
@@ -85,3 +85,30 @@ Resources:
...
+  TCNJ-ELSA-CE2:
+    ID: 
+    Active: true
+    Description: The Hosted CE2 serving TCNJ-ELSA
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Marco Mascheroni
+          ID: 030408ab932e143859b5f97a2d1c9e30ba2a9f0d
+        Secondary:
```
Comment (brianhlin) [inline topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:97]:
```suggestion
```

File: topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml [hunk 3]
```diff
@@ -85,3 +85,30 @@ Resources:
...
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+      Security Contact:
+        Primary:
+          Name: Marco Mascheroni
+          ID: 030408ab932e143859b5f97a2d1c9e30ba2a9f0d
+        Secondary:
```
Comment (brianhlin) [inline topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:104]:
```suggestion
```

**Review by brianhlin — COMMENTED** [review]
Ah but we probably shouldn't have Marco listed as the contact

**Pull request merged**